includes/Dashboard/Templates/Settings.php

```diff
@@ -647,4 +762,75 @@ function get_dokan_categories() {
 
         return apply_filters( 'dokan_category', $dokan_category );
     }
+
+    /**
+     * Get proper heading for payments of vendor dashboard payment settings
+     *
+     * @since DOKAN_SINCE
+     *
+     * @param string $slug
+     * @param string $heading
+     *
+     * @return string
+     */
+    private function get_payment_heading( $slug, $heading ) {
+        switch ( $slug ) {
+            case 'bank':
+            case 'bank/edit':
+                $heading = __( 'Bank Account Settings', 'dokan-lite' );
+                break;
+
+            case 'paypal':
+            case 'paypal/edit':
+                $heading = __( 'Paypal Settings', 'dokan-lite' );
+                break;
+        }
+
+        /**
+         * To allow new payment extension give their own heading
+         *
+         * @since DOKAN_SINCE
+         *
+         * @param string $heading previous heading
+         */
+        $heading = apply_filters( "dokan_withdraw_method_settings_title", $heading, $slug );
+
+        return $heading;
+    }
+
+    /**
+     * Get unused payment methods
+     *
+     * @since DOKAN_SINCE
+     *
+     * @param array $methods           All methods
+     * @param array $profile_methods   Used Methods
+     * @param array $mis_match_methods The list of methods which has different key and name but the name is substring of key
+     *
+     * @return array
+     */
+    private function dokan_get_unused_payment_methods( $methods, $profile_methods, $mis_match_methods ) {
```

Comment:
`dokan_` prefix isn't needed for class method and try to improve this method. See the previous review comment on this method.